### PR TITLE
[FLOC-2677] add Optimizely to docs template

### PR DESCRIPTION
**Pull request by msh100**
Optimizely snippet.

Fixes FLOC-2677

http://build.clusterhq.com/results/docs/docs-optimizely-FLOC-2677/build-9236/ is running a test at the moment, I changed the text to 

> You are reading an in-development version of the documentation. Some of the functionality may not work as expected. Things may break.

In the warning I added "Things may break"

### CODE REVIEW

File: docs/_themes/clusterhq/layout_docs.html [hunk 1]
```diff
@@ -204,6 +204,9 @@
         </script>
     {% endif %}
 
+    <!-- Optimizely -->
+    <script src="https://cdn.optimizely.com/js/2309470232.js"></script>
```
Comment (jml) [inline docs/_themes/clusterhq/layout_docs.html]: Could you please add a comment saying where this URL comes from?

**Comment by jml** [comment]
Please add the comment and merge when ready

**Comment by msh100** [comment]
@jml are you happy with that?

**Comment by adamtheturtle** [comment]
@msh100 asked me whether this is OK because @jml is away. I've said yes, merge when the build passes.

**Pull request merged**